editor/export/editor_export_platform.cpp

```diff
@@ -1444,32 +1418,26 @@ Error EditorExportPlatform::export_project_files(const Ref<EditorExportPreset> &
 					resolve_platform_feature_priorities(p_preset, remap_features);
 				}
 
-				err = OK;
-
 				for (const String &F : remaps) {
 					String remap = F;
 					if (remap == "path") {
 						String remapped_path = config->get_value("remap", remap);
 						Vector<uint8_t> array = FileAccess::get_file_as_bytes(remapped_path);
-						err = save_proxy.save_file(p_udata, remapped_path, array, idx, total, enc_in_filters, enc_ex_filters, key, seed);
+						RETURN_IF_ERR(save_proxy.save_file(p_udata, remapped_path, array, idx, total, enc_in_filters, enc_ex_filters, key, seed));
```

Comment:
This changes the behavior of the loop, which should be investigated, but this changes the behavior, so should be excluded